src/client/pydaos/raw/daos_cref.py

```diff
@@ -99,6 +99,31 @@ class PoolInfo(ctypes.Structure):
                 ("pi_space", PoolSpace),
                 ("pi_rebuild_st", RebuildStatus)]
 
+
+class DaosPropertyEntry(ctypes.Structure):
```

Comment:
(pylint-missing-docstring) Missing class docstring
(pylint-too-few-public-methods) Too few public methods (0/2)

Reply:
Known problem with this file.